modules/smart-agent_kubernetes-workloads-count/conf/01-kubernetes-workloads-count.yaml

```diff
@@ -0,0 +1,24 @@
+module: "Kubernetes"
+name: "Workloads Count"
+filtering: "not filter('kubernetes_namespace','ara') and not filter('kubernetes_namespace','bastions') and not filter('kubernetes_namespace','gitlab-runner') and not filter('kubernetes_namespace','logging') and not filter('kubernetes_namespace','monitoring') and not filter('kubernetes_namespace','ingress-nginx') and not filter('kubernetes_namespace','kube-system') and not filter('kubernetes_namespace','kubernetes-replicator')"
+aggregation: ".sum(by=['kubernetes_name','kubernetes_namespace']).count()"
+transformation: ".min(over='30m')"
+value_unit: "entries"
```

Comment:
I doubt "entries" is correct. "records" is probably what you look for but I am not sure it is useful to set a unit here a simple "count".